src/pages/index.js

```diff
@@ -1,38 +1,40 @@
 // ***********************************   DECLARING VARIABLES   ************************************************************
 //IMPORTS
 import './index.css';
-import {initialCards, editButton, profileNameClass, profileDescriptionClass, addPlaceButton, cardModalForm, profileModalForm, cardContainerClass} from '../utils/constants.js';
-import Section from '../components/Section.js';
-import {makePopUpVisible, makeAddPlacePopUpVisible, createNewUserInfo, createFormValidator, createPopupWithForm, createPopupWithImage, createNewCard} from '../utils/utils.js';
+import {editButton, profileNameClass, profileDescriptionClass, addPlaceButton, cardModalForm, profileModalForm, avatarModalForm, avatarButton, profileModal, cardModal, avatarModal} from '../utils/constants.js';
+import {makePopUpVisible, makeAddPlacePopUpVisible, createNewUserInfo, createFormValidator, createPopupWithForm, createPopupWithImage, makeAvatarPopUpVisible} from '../utils/utils.js';
+import Api from '../components/Api.js'
 
 
+const api = new Api();
+api.loadUserInfo();
+api.loadCards();
```

Comment:
You need to combine the requests of the user's info and the cards together in `Promise.all` to handle the 2 responses here in  `index.js`

More info is here: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise/all

```js

Promise.all([getUserInfo(), getCards()])
// destructure the response
  .then(([userData, cards]) => {
      // set all the data
  })
  .catch(err => {
    // catch possible errors
  });

```